### ISSUE TITLE
ToolboxBitmapAttributeTests.Ctor_FileName test failed on netfx

### ISSUE THREAD

**Issue by stephentoub** [dotnet/corefx#27361]
https://ci3.dot.net/job/dotnet_corefx/job/master/job/windows-TGroup_netfx+CGroup_Release+AGroup_x86+TestOuter_false_prtest/8667/consoleText
```
      System.Drawing.Tests.ToolboxBitmapAttributeTests.Ctor_FileName(fileName: "D:\\j\\workspace\\windows-TGrou---2a8f9c29\\bin\\t"..., size: {Width=16, Height=16}) [FAIL]
        System.ArgumentException : Parameter is not valid.
        Stack Trace:
             at System.Drawing.Image.get_Width()
             at System.Drawing.Image.get_Size()
             at System.Drawing.Tests.ToolboxBitmapAttributeTests.Ctor_FileName(String fileName, Size size)
```

**Comment by stephentoub** [comment]
Failed again here:
https://ci3.dot.net/job/dotnet_corefx/job/master/job/windows-TGroup_netfx+CGroup_Release+AGroup_x86+TestOuter_false_prtest/9306/consoleText
```
      System.Drawing.Tests.ToolboxBitmapAttributeTests.Ctor_FileName(fileName: "D:\\j\\workspace\\windows-TGrou---2a8f9c29\\bin\\t"..., size: {Width=16, Height=16}) [FAIL]
        System.ArgumentException : Parameter is not valid.
        Stack Trace:
             at System.Drawing.Image.get_Width()
             at System.Drawing.Image.get_Size()
             at System.Drawing.Tests.ToolboxBitmapAttributeTests.Ctor_FileName(String fileName, Size size)
```

**Comment by weshaggard** [comment]
I've been seeing the opposite race condition on core runs which I believe I've uncovered and address with https://github.com/dotnet/corefx/pull/29148/commits/1e24a59325b2bdf5050db70923aca4299dab9ba1.  After debugging through the tests I finally figured out we have a race condition around the creation these dummy images. I suspect this bug exists on .NET Framework as well, so that leg may still fail and we might need to update the tests accordingly.
